scripts/ci/crates.py

```diff
@@ -429,7 +429,8 @@ def publish_crate(crate: Crate, token: str, version: str, env: dict[str, Any]) -
     retry_attempts = 5
     while True:
         try:
-            cargo(f"publish --quiet --token {token}", cwd=crate.path, env=env, dry_run=False, capture=True)
+            # We added --locked here hoping that `cargo publish` would use the included `Cargo.lock` file, but it doesn't.
```

Comment:
This ultimately didn't help, but it shouldn't hurt either. Keeping it at least makes it obvious next time, that adding it isn't the solution